gnovm/stdlibs/std/banker.go

```diff
@@ -88,10 +88,17 @@ func X_bankerTotalCoin(m *gno.Machine, bt uint8, denom string) int64 {
 
 func X_bankerIssueCoin(m *gno.Machine, bt uint8, addr string, denom string, amount int64) {
 	// gno checks for bt == RealmIssue
-	m.Context.(ExecContext).Banker.IssueCoin(crypto.Bech32Address(addr), denom, amount)
+
+	// Similar to ibc spec
+	// ibc_denom := 'ibc/' + hash('path' + 'base_denom')
+	// gno_realm_denom := 'realm/' + 'pkg_path' + '/' + 'base_denom'
+	newDenom := "realm/" + m.Realm.Path + "/" + denom
```

Comment:
If we have the realm `gno.land/r/demo/a/b`, it can issue coin `b`, but `gno.land/r/demo/a` can also issue coin `b/c`, which would then be the same.

Can we forbid `/` in denoms? (before joining them into the path). I don't know if they're commonly used, but it would seem confusing to me.

Alternatively, use a different separator here, like `#`.

It also seems reasonable to me to limit the length of denominations, as we currently are. At the current stage, we could have denoms up to 100 chars long. The `[3,16]` boundary currently in place seems reasonable.